pyproject.toml

```diff
@@ -71,3 +71,33 @@ setuptools = "pkg_resources"
 CacheControl = "https://raw.githubusercontent.com/ionrock/cachecontrol/v0.12.6/LICENSE.txt"
 distlib = "https://bitbucket.org/pypa/distlib/raw/master/LICENSE.txt"
 webencodings = "https://github.com/SimonSapin/python-webencodings/raw/master/LICENSE"
+
+[tool.ruff]
+exclude = [
+    "./build",
+    ".nox",
+    ".tox",
```

Comment:
its by default excluded https://beta.ruff.rs/docs/settings/#exclude
it should be `extend-exclude`

Reply:
Thanks. I got this by running [flake8-to-ruff](https://pypi.org/project/flake8-to-ruff/) and didn't change much from what it suggested. I'll fix this.